chordspy/csvplotter.py

```
@@ -99,7 +99,10 @@ def plot_data(self):
         )
         fig.show()               # Display the plot in a new window
 
-if __name__ == "__main__":
+def main():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)